plugins/com.google.cloud.tools.eclipse.projectselector.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Tests for Cloud Tools for Eclipse Project Selector Widget
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.projectselector.test
+Bundle-Vendor: Google Inc.
+Bundle-Version: 0.1.0.qualifier
+Fragment-Host: com.google.cloud.tools.eclipse.projectselector
+Require-Bundle: com.google.guava;bundle-version="[20.0.0,21.0.0)",
```

Comment:
can you import the specific packages rather than requiring the bundle? (may not work, but worth trying.)